docs/devcontainer-example/Dockerfile.llvm-project

```diff
@@ -0,0 +1,21 @@
+# By default, use ubuntu:focal
+ARG BASE_IMAGE="onnxmlirczar/onnx-mlir-llvm-static:amd64"
+FROM ${BASE_IMAGE}
+
+RUN apt-get update
+
+# 2) Install optional packages, comment/uncomment/add as you see fit.
+RUN apt-get install -y vim bash valgrind clang-format lsb-release \
+    wget software-properties-common ssh-client gdb
+RUN python3 -m pip install -U pip && pip install commitizen
+# Install clang
+RUN wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh \
+    && ./llvm.sh 15 "" && rm llvm.sh
+RUN ln -s /usr/bin/llvm-symbolizer-15 /usr/bin/llvm-symbolizer \
+    && ln -s /usr/bin/clang-15 /usr/bin/clang \
+    && ln -s /usr/bin/clang++-15 /usr/bin/clang++ \
```

Comment:
Wondering about `lld` and also if `-DCMAKE_EXE_LINKER_FLAGS_INIT="-fuse-ld=lld" -DCMAKE_MODULE_LINKER_FLAGS_INIT="-fuse-ld=lld" -DCMAKE_SHARED_LINKER_FLAGS_INIT="-fuse-ld=lld"` is then needed if you want to use it as the linker?

Reply:
clang didn't work out, I delete references to that. If we find a way later we can add it back